Tests/Unit/Commenting/FunctionCommentSniffTest.phptest

```diff
@@ -1,7 +1,7 @@
 <?php
 
 /*
- *   This file is part of the ONGR package.
+ * This file is part of the ONGR package.
```

Comment:
No, it wasn't. I inserted spaces myself and I forgot to remove it before the first commit. 
But there is a problem with block comments, you can add spaces almost everywhere and it will be ignored(the same problem was in v1.0)